yt_dlp/extractor/reverbnation.py

```diff
@@ -46,6 +48,64 @@ def _real_extract(self, url):
             'uploader': api_res.get('artist', {}).get('name'),
             'uploader_id': str_or_none(api_res.get('artist', {}).get('id')),
             'thumbnails': thumbnails,
+            'duration': api_res.get('duration'),
+            'tbr': api_res.get('bitrate'),
             'ext': 'mp3',
             'vcodec': 'none',
         }
+
+
+class ReverbNationArtistIE(InfoExtractor):
+    IE_NAME = 'reverbnation:artist'
+    _VALID_URL = r'^https?://(?:www\.)?reverbnation\.com/(?P<id>[\w-]+)(?:/songs)?$'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?reverbnation\.com/(?P<id>[\w-]+)(?:/songs)?$'
```
see https://github.com/yt-dlp/yt-dlp/pull/10791

Reply:
Thanks, updated